learningresources/api.py

```diff
@@ -301,3 +305,44 @@ def import_static_assets(course, path):
                 name = join(root, name).replace(path + sep, '', 1)
                 django_file.name = name
                 create_static_asset(course.id, django_file)
+
+
+def join_description_paths(*args):
+    """
+    Helper function to format the description path.
+    Args:
+        list of dpath (unicode): description path
+    Returns:
+        formatted dpath (unicode)
```

Comment:
For the argument you should use the format `args (type here): description path`. Whatever you think is sensible for the type is fine with me

The return section should be `unicode: Formatted dpath`